### PR TITLE
Fixed links to blank target, company name etc. [Fixes #22][Fixes #26][Fixes#27]

### LINKED ISSUE #22: links that go off site should be target=_blank

### LINKED ISSUE #26: "creators" link should be an actual page
http://screencast.com/t/QIvThuZc6Va

Right now it goes straight to github. It should go to daftlabs.com

### PR DESCRIPTION
**Pull request by avophile**
[Fixes #22][Fixes #26][Fixes#27]

### CODE REVIEW

File: views/about.lavender [hunk 1]
```diff
@@ -20,22 +20,25 @@ block content
...
-          h1 Daft Labs
+          h1 daft labs
           p
-            | Daft Labs was founded in 2013, slowly bringing together a team of hodge 
-            | podge people in both front end and back end development.  
-          a(href="http://www.daftlabs.com", target="_blank") Come Visit Us.
+            a(href='http://daftlabs.com/', target='_blank') daft labs
+            |  was founded in 2013, bringing together a hodge podge
+            | team of front and back end developers.  
+          a(href='http://www.daftlabs.com', target='_blank') Come Visit Us.
```
Comment (sugarPhlox) [inline views/about.lavender:39]: There's a double link here (linking to daft labs at the beginning and end of the section.) Don't know if that's really necessary.

Comment (avophile) [inline views/about.lavender:39]: One wants to be inviting! : )

**Comment by mowchan** [comment]
:cake:

**Comment by sugarPhlox** [comment]
:cake:

**Pull request merged**